integration/ig/k8s/trace_dns_test.go

```diff
@@ -164,3 +165,95 @@ func TestTraceDns(t *testing.T) {
 
 	RunTestSteps(commands, t, WithCbBeforeCleanup(PrintLogsFn(ns)))
 }
+
+func TestTraceDnsHost(t *testing.T) {
+	t.Parallel()
+	ns := GenerateTestNamespaceName("test-trace-dns")
+
+	commandsPreTest := []*Command{
+		CreateTestNamespaceCommand(ns),
+		PodCommand("dnstester", *dnsTesterImage, ns, "", ""),
+		WaitUntilPodReadyCommand(ns, "dnstester"),
+	}
+
+	RunTestSteps(commandsPreTest, t)
+	dnsServer, err := GetTestPodIP(ns, "dnstester")
+	if err != nil {
+		t.Fatalf("failed to get pod ip: %v", err)
+	}
+
+	traceDNSCmd := &Command{
+		Name:         "TraceDnsHost",
+		Cmd:          "ig trace dns -o json --host",
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntries := []*dnsTypes.Event{
+				{
+					Event: eventtypes.Event{
+						Type: eventtypes.NORMAL,
+					},
+					Comm:       "isc-worker0000",
```

Comment:
Why do we need to use `isc-worker0000` and why isn't nslookup reported there? Is this a bug on the enrichment?

Reply:
nslookup has several threads and one of them is called `isc-worker0000`:
```
$ cat /proc/$(pidof nslookup)/task/*/comm
nslookup
isc-net-0000
isc-timer
isc-socket-0
```

But it does dependent on the nslookup binary. If you use the busybox nslookup, it is not multithreaded. If you use the host nslookup, depending on the Linux distribution, it might be.

The socket enricher currently looks at the `comm` of the thread and not the process because it uses `bpf_get_current_comm`. Except for initial sockets, where we cannot distinguish the threads that created the socket: in that case we take the `comm` from the main thread.

Reply:
Thanks a lot for the explanation. It'd be useful for include this on a comment on the test.

Reply:
I'd prefer not to normalize (remove) fields from the testing. Each time we "normalize" a field we reduce our test coverage. Let's keep `isc-worker0000`, if this starts causing issues we can easily normalize it.